src/Evolution/Initialization/Evolution.hpp

```diff
@@ -49,33 +51,39 @@ namespace Initialization {
 /// \note HistoryEvolvedVariables is allocated, but needs to be initialized
 template <typename System>
 struct Evolution {
+  using initialization_option_tags =
+      tmpl::list<Tags::InitialTime, Tags::InitialTimeDelta,
+                 Tags::InitialSlabSize>;
+
```

Comment:
everything but the option tags and `apply` should be private

Reply:
We've never be strict on this, so I'm leaving it.